### DISCUSSION TITLE
Airsdk 51 Android File Read & Write: I request professional support

### DISCUSSION THREAD

**Discussion by SerdarYildiz06** [airsdk/Adobe-Runtime-Support#3528] [General]
Hello, I have an e-book application that I developed with airsdk. I was using Airsdk version 31.0.0.96 and I had no problems on both iOS and Android. I upgraded my airsdk version to compile sdk 34 and my current airsdk version is 51.1. While there is no problem in the iOS application, I cannot open the books in the Android application after downloading them. I think I have done something missing in the file reading and writing section on the Android part. I request professional support

**Comment by raresn** [comment]
Hey - did you try to connect your phone to a PC and do a debug? People might be able to help if you give a more specific error.

**Reply by SerdarYildiz06** [reply]
We couldn't try it because the system works like this, I add the relevant parameters to the animate project, it gives me apk and appbunde as output, I can't debug it directly.

**Reply by raresn** [reply]
You need to add parameters for that apk to be debuggable - and then connect a phone to the pc and see what error it gives you.
